packages/core-components/api-report.md

```diff
@@ -2404,6 +2404,14 @@ export function TrendLine(
     },
 ): JSX.Element | null;
 
+// Warning: (ae-missing-release-tag) "useContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
```

Comment:
would it be nice having this documented? just a short explanation

Reply:
Thanks, updated now, hope it makes sense